src/constants.ts

```diff
@@ -6,6 +6,7 @@ export const spaBasePath = `${window.spaBase}${basePath}`;
 export const OPENMRS_FHIR_PREFIX = "http://fhir.openmrs.org";
 export const OPENMRS_FHIR_EXT_PREFIX = OPENMRS_FHIR_PREFIX + "/ext";
 export const OPENMRS_FHIR_EXT_MEDICINE = OPENMRS_FHIR_EXT_PREFIX + "/medicine";
+export const OPENMRS_FHIR_EXT_RECORDED = OPENMRS_FHIR_EXT_PREFIX + "/recorded";
```

Comment:
We've recently updated the Medication Dispense resource in FHIR to have a new "recorded" extension, see FM2-563